Scheduler/observing_blocks_maintel/star-tracker-sit-area.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "SIT-Imaging",
+    "program": "SIT:Area",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                    "ra": "$ra", 
+                    "dec": "$dec"
+                },
+                "target_name": "$name",
+                "rot_value": 0.0,
+                "rot_type": "Sky",
```

Comment:
We are not going to be using the rotator and rot=0/rot_type=Sky is not observable everywhere. Maybe use PhysicalSky? Or just Physical?